app/assets/stylesheets/custom.css.scss

```diff
@@ -166,6 +165,18 @@ table td {
   margin-right: 10px;
 }
 
+.dropdown-menu li:hover .sub-menu {
+    visibility: visible;
+}
+
+.dropdown:hover .dropdown-menu {
+    display: block;
+}
+
+.account{
+    float:left
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Colon after property should be followed by one space<br>Declaration should be terminated by a semicolon